Lib/asyncio/tasks.py

```diff
@@ -25,6 +25,8 @@
 from . import futures
 from .coroutines import _is_coroutine
 
+_SENTINEL = object()
```

Comment:
Please use `enum` trick, as we do in [aiohttp.helpers](https://github.com/aio-libs/aiohttp/blob/master/aiohttp/helpers.py#L72-L73)

The PEP for sentinels is not accepted yes, enums are forward-compatible at least.

stdlib has no embedded type hints but still.

Reply:
The enum approach seemed like a bit of a hack for static typing, so didn't seem useful when there are no type hints. But, I don't mind changing it.